.github/workflows/build.yml

```diff
@@ -0,0 +1,31 @@
+name: Build
+
+on: [ push, pull_request, workflow_dispatch ]
```

Comment:
What did you have in mind with `workflow_dispatch` : https://docs.github.com/en/actions/reference/events-that-trigger-workflows#workflow_dispatch ?

Reply:
After being merged into main, will have the ability to run workflow without push or PR. This is useful for advanced integration testing. Such as if Vitro default branch changes, you can trigger a build CI of VIVO without a push or PR.